zoo-project/zoo-kernel/zoo_service_loader.c

```diff
@@ -3900,7 +3908,7 @@ runRequest (map ** inputs)
 #endif
 	if(cIdentifier!=NULL)
 	  addToMap(request_inputs,"Identifier",cIdentifier);
-	if(fetchService(zooRegistry,m,&s1,request_inputs,ntmp,cIdentifier,localPrintExceptionJ)!=0){
+	if(fetchService(zooRegistry,m,&s1,request_inputs,conf_dir_,cIdentifier,localPrintExceptionJ)!=0){
```

Comment:
This change is not linked directly to the support of the logs on the stderr but it is a small bug fix. 
In fact without this change the parameter "servicePath" is not used to define a different path to search for services rather than using the ZOO-Kernel directory.

This change is also mandatory to allow us to move to docker hosting since we have to mount a folder from a volume that stores our python services different than the folder that stores the ZOO-Kernel.

Without this change the lines 2693 to 2705 of the same file are never used.

Reply:
Thanks for the clarification.